lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -2,22 +2,21 @@ const ChangeTypes = require('../../ChangeTypes');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// If `node` is a product of terms where some are fractions (but none are
-// polynomial terms), multiplies them together.
+// If `node` is a product of terms where:
+// 1) at least one is a fraction
+// 2) either none are polynomial terms, OR
+//    at least one has a symbol in the denominator
+// then multiply them together.
```

Comment:
No, I don't think so, and I'm actually not positive why this is the case! Is it because of your comment on the issue here: "there are some cases where we don't want to multiply fractions with other things, one of them is 9/2 * x -> 9x / 2 (since this doesn't change anything and could be considered less readable)" ?